src/test/rmc/Options/check_tests.rs

```diff
@@ -4,7 +4,7 @@
 // Check that we can verify test harnesses using the --tests argument.
 // Note: We need to provide the compile-flags because compile test runs rustc directly and via rmc.
 
-// compile-flags: --test
+// compile-flags: --test --C panic=abort
```

Comment:
Is this needed?

Reply:
Let me double check. I think you are correct though.

Reply:
was it necessary?